tooling/lint-package-names.js

```diff
@@ -0,0 +1,53 @@
+// const shelljs = require(`shelljs`);
+const fs = require(`fs`);
+
+const modules = [];
+
+
+// JS equivelent of "set -e" bash command
+// for (var envVariable in process.env ) {
+//     shelljs.exec(`echo ${envVariable}`);
+// }
+
+// Grab the module names of each package in the packages/node_modules 
+// and package/node_modules/@ciscospark directories containing the ciscospark substring
+// Using a Promise to excecute async functions in a sync manner to pass linting tests
+new Promise((resolve, reject) => {
```

Comment:
Implement this file in `tooling/lib` and `tooling/commands`. There are already helpers for enumerating package names and reading package.jsons